src/sst/core/rng/rng.cc

```diff
@@ -9,7 +9,6 @@
 // information, see the LICENSE file in the top level directory of the
 // distribution.
 
-
 #include "sst_config.h"
+#include "rng.h"
 #include "sstrng.h"
```

Comment:
Do we still need to include sstrng.h in rng.cc?

Reply:
Yes. We are moving to `rng.h` but current API will refer to `sstrng.h` so we are providing both to be compatible. We can decide how to move forward with class deprecation over time using this approach.

Reply:
Actually, a closer look shows that this file (rng.cc) isn't strictly needed at all because the rng.h header file is included in some other .cc files and there are no additional implementation details here (confirmed by a quick build without the file).  However, we can make a decision on whether to keep it or not later.

Reply:
The point it to include both so we can remove `sstrng.h` once deprecation is decided.